dist/md-date-time.css

```diff
@@ -11,6 +11,13 @@
 .visuallyhidden {
   visibility: hidden;
 }
+.md-button {
+  padding: 6px;
+  margin: 0;
+  min-width: initial;
+  line-height: inherit;
+  font-size: inherit;
+}
```

Comment:
i think you might want to wrap this in .time-date? you are altering all md-buttons here